pkgs/tools/misc/empty/default.nix

```diff
@@ -0,0 +1,48 @@
+{ fetchzip, lib, stdenv, which }:
+
+let
+  pname = "empty";
+  version = "0.6.21b";
+in stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchzip {
+    url = "mirror://sourceforge/${pname}/${pname}/${pname}-${version}.tgz";
+    sha256 = "1rkixh2byr70pdxrwr4lj1ckh191rjny1m5xbjsa7nqw1fw6c2xs";
+    stripRoot = false;
+  };
+
+  nativeBuildInputs = [ which ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  # Allow installation prefix to be overridden, and fix the CC parameter order.
+  prePatch = ''
+    substituteInPlace Makefile \
+      --replace 'PREFIX = ' 'PREFIX ?= ' \
```

Comment:
Have you tested the package without this line? I suspect the make variant which nixpkgs uses doesn't require it.

Reply:
Thanks: that's good to know!